### PR TITLE
Allow for simultaneous requests on multiple entities

### PR DESCRIPTION
**Pull request by cramertj**
Also moved some things from Java to Scala in the process.

### CODE REVIEW

File: project/BalboaAdmin.scala [hunk 1]
```diff
@@ -7,11 +7,12 @@ object BalboaAdmin {
   lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings ++ Seq(
     mainClass in sbtassembly.AssemblyKeys.assembly := Some("com.socrata.balboa.admin.BalboaAdmin"),
     libraryDependencies <++= scalaVersion { libraries(_) },
-    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 100
+    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 75
```
Comment (raganw) [inline project/BalboaAdmin.scala]: heh

**Comment by raganw** [comment]
I think this looks good, but I'm going to check it out again tomorrow with fresh eyes.

File: project/BalboaAdmin.scala (hunk 1 above)
Comment (cramertj) [inline project/BalboaAdmin.scala]: 😢

**Comment by raganw** [comment]
Looks good to me :shipit:

**Pull request merged**